transformer_engine/pytorch/cpp_extensions/fused_attn.py

```diff
@@ -109,17 +143,18 @@ def fused_attn_fwd_qkvpacked(
     cu_seqlens: torch.Tensor,
     qkv: torch.Tensor,
     qkv_dtype: tex.DType,
-    bias: torch.Tensor = None,
+    fused_attention_backend: tex.NVTE_Fused_Attn_Backend,
+    attn_bias: torch.Tensor = None,
     d_scale_qkv: torch.Tensor = None,
     q_scale_s: torch.Tensor = None,
     q_scale_o: torch.Tensor = None,
     amax_s: torch.Tensor = None,
     amax_o: torch.Tensor = None,
     attn_scale: float = None,
     dropout: float = 0.0,
-    set_zero: bool = True,
+    fast_zero_fill: bool = True,
```

Comment:
cuDNN provides zero fill feature in 8.9.1, and this is integrated in [fmha_v1](https://github.com/NVIDIA/TransformerEngine/blob/main/transformer_engine/common/fused_attn/fused_attn_fp16_bf16_max_seqlen_512.cu#L533) and enabled automatically. Do you have a plan to integrate this for fmha_v2/flash_attn?

Reply:
cuDNN doesn't support `.setPaddingValue(0.0f)` for fmha_v2 as I checked with the team. So we will need to do the zero resets using either the framework's `.fill_(0)` or the fast [`mha_fill`](https://github.com/cyanguwa/TransformerEngine/blob/71a1b9502de70885a8864f8bfccbcfcdc71d7802/transformer_engine/pytorch/csrc/extensions.cu#LL47C24-L47C24) method from MLPerf.